Logger.Tests/FileLoggerTests.cs

```diff
@@ -1,10 +1,47 @@
-﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+﻿
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
+using System.Linq;
+
 
 namespace Logger.Tests
 {
     [TestClass]
     public class FileLoggerTests
     {
+        [TestMethod]
+        public void Getpath_Success()
```

Comment:
This is a minor thing, but in general you want you unit test name to reflect the method that is being tested. In this case something more like: 
```suggestion
        public void FileLogger_Constructor_CreatesFile()
```